pages/your_tasks/data_discoverability.md

```diff
@@ -0,0 +1,81 @@
+
+### Solutions
+
+{% tool "beacon" %}, developed through the Global Alliance for Genomics and Health (GA4GH) Discovery workstream, and with substantial support from ELIXIR, serves as a data discovery protocol and specification defining an open standard for discovering genomic and phenoclinic data in biomedical research and clinical applications.
+
+The latest version, {% tool "beacon-v2" %}, introduced expanded query options, enabling the retrieval of biological or technical (meta)data through filters defined via CURIEs. This includes, but not limited to, parameters like phenotypes, disease codes, sex, or age, providing researchers with a nuanced approach to data inquiries.
+
+Beacon v2 is organized in two main blocks: the Beacon Framework and the Beacon Model. 
+The Framework defines the format for the requests and responses, whereas the Model defines the structure of the biological data response.
+
+This dual-system approach not only broadens the scope for diverse Models – using different domains like images, pathogens, or infectious diseases– but also reinforces the adaptability of the Framework. The overall function of these components is to provide the instructions to design a REST API that could be implemented as a stand-alone product or, preferably, extending existing data management solutions. 
+
+Consequently, the 'beaconized' data represents a significant enhancement in data discoverability with minimal risks. 
+
+Currently there are two ways to implement a Beacon:
+
+* On top of existing tools (API): Targeted to those organizations equipped with well-organized and structured data housed in databases, whether SQL or NoSQL, and possess the necessary resources and expertise to interpret and implement the Beacon v2 specification and construct an API on top of an existing tool.
+
+  {% include image.html file="beacon-api.JPG" inline=true caption="Figure 1. Beacon API funcionality ([Source](https://docs.genomebeacons.org/implementations-options/))" alt="Beacon API" max-width="30em"%}
+
+* {% tool "beacon-ri" %}: Beacon v2 Reference Implementation, B2RI, is an out-of-the-box example implementation of the Beacon v2 protocol. It is an open-source toolkit based on Python programming language and consists of tools for loading metadata, e.g.phenotypic data, from a CSV file and genomic variants from a VCF file into a MongoDB database. It also features the Beacon query engine (REST API) and comes bundled with an example dataset (CINECA synthetic cohort EUROPE UK1) comprising synthetic data.
+
+  {% include image.html file="beacon-ri.JPG" inline=true caption="Figure 2. Beacon RI funcionality. ([Source](https://docs.genomebeacons.org/implementations-options/))" alt="Beacon RI" max-width="30em" %}
+
+Documentation: https://b2ri-documentation.readthedocs.io/en/latest/ 
+GitHub Repository: https://github.com/EGA-archive/beacon2-ri-tools-v2 
```

Comment:
Don't know if I like the way this is listed. Any other opinions?

Reply:
Yeah it's hard to tell what are the two ways - I think what might help is to add to the text what is the documentation and GitHub repo for? e.g. simply "Documentation for B2RI: ". What do  you think, @lauportell?

Reply:
done